apps/ewallet_db/test/ewallet_db/key_test.exs

```diff
@@ -90,22 +90,37 @@ defmodule EWalletDB.KeyTest do
     end
   end
 
-  describe "update/2" do
-    test_update_field_ok(Key, :expired, insert(:admin), false, true)
-    test_update_ignores_changing(Key, :access_key)
+  describe "enable_or_disable/2" do
```

Comment:
Can you add a test that disabling it twice doesn't make it enabled?

Just in case someone turns it into a toggle in the future.